<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head><title>R: Calls Copy Number Variants from Targeted Sequence Data</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
<link rel="stylesheet" type="text/css" href="R.css" />
</head><body><div class="container">
<h1> Calls Copy Number Variants from Targeted Sequence Data
<img class="toplogo" src="../../../doc/html/Rlogo.svg" alt="[R logo]" />
</h1>
<hr/>
<div style="text-align: center;">
<a href="../../../doc/html/packages.html"><img class="arrow" src="../../../doc/html/left.jpg" alt="[Up]" /></a>
<a href="../../../doc/html/index.html"><img class="arrow" src="../../../doc/html/up.jpg" alt="[Top]" /></a>
</div><h2>Documentation for package &lsquo;ExomeDepth&rsquo; version 1.1.15</h2>

<ul><li><a href="../DESCRIPTION">DESCRIPTION file</a>.</li>
<li><a href="../doc/index.html">User guides, package vignettes and other documentation.</a></li>
</ul>

<h2>Help Pages</h2>


<table width="100%">
<tr><td style="width: 25%;"><a href="AnnotateExtra-ExomeDepth-method.html">AnnotateExtra</a></td>
<td>AnnotateExtra</td></tr>
<tr><td style="width: 25%;"><a href="AnnotateExtra-ExomeDepth-method.html">AnnotateExtra-method</a></td>
<td>AnnotateExtra</td></tr>
<tr><td style="width: 25%;"><a href="CallCNVs-ExomeDepth-method.html">CallCNVs</a></td>
<td>CallCNVs</td></tr>
<tr><td style="width: 25%;"><a href="CallCNVs-ExomeDepth-method.html">CallCNVs-method</a></td>
<td>CallCNVs</td></tr>
<tr><td style="width: 25%;"><a href="Conrad.hg19.common.CNVs.html">Conrad.hg19.common.CNVs</a></td>
<td>Conrad et al common CNVs</td></tr>
<tr><td style="width: 25%;"><a href="count.everted.reads.html">count.everted.reads</a></td>
<td>Count the number of everted reads for a set of BAM files.</td></tr>
<tr><td style="width: 25%;"><a href="countBam.everted.html">countBam.everted</a></td>
<td>Counts everted reads from a single BAM file</td></tr>
<tr><td style="width: 25%;"><a href="countBamInGRanges.exomeDepth.html">countBamInGRanges.exomeDepth</a></td>
<td>Compute read count data from BAM files.</td></tr>
<tr><td style="width: 25%;"><a href="C_hmm.html">C_hmm</a></td>
<td>C_hmm</td></tr>
<tr><td style="width: 25%;"><a href="ExomeCount.html">ExomeCount</a></td>
<td>Example dataset for ExomeDepth</td></tr>
<tr><td style="width: 25%;"><a href="ExomeDepth-class.html">ExomeDepth-class</a></td>
<td>Class 'ExomeDepth'</td></tr>
<tr><td style="width: 25%;"><a href="exons.hg19.html">exons.hg19</a></td>
<td>Positions of exons on build hg19 of the human genome</td></tr>
<tr><td style="width: 25%;"><a href="exons.hg19.X.html">exons.hg19.X</a></td>
<td>Positions of exons on build hg19 of the human genome and on chromosome X</td></tr>
<tr><td style="width: 25%;"><a href="genes.hg19.html">genes.hg19</a></td>
<td>Positions of genes on build hg19 of the human genome</td></tr>
<tr><td style="width: 25%;"><a href="get.power.betabinom.html">get.power.betabinom</a></td>
<td>Estimate the power to compare two beta-binomial distributions.</td></tr>
<tr><td style="width: 25%;"><a href="getBamCounts.html">getBamCounts</a></td>
<td>Get count data for multiple exomes</td></tr>
<tr><td style="width: 25%;"><a href="get_loglike_matrix.html">get_loglike_matrix</a></td>
<td>get_loglike_matrix</td></tr>
<tr><td style="width: 25%;"><a href="initialize-ExomeDepth-method.html">initialize-method</a></td>
<td>ExomeDepth initialization tool</td></tr>
<tr><td style="width: 25%;"><a href="plot-methods.html">plot-method</a></td>
<td>Plotting function for ExomeDepth objects</td></tr>
<tr><td style="width: 25%;"><a href="plot-methods.html">plot-methods</a></td>
<td>Plotting function for ExomeDepth objects</td></tr>
<tr><td style="width: 25%;"><a href="plot-methods.html">plot.ExomeDepth</a></td>
<td>Plotting function for ExomeDepth objects</td></tr>
<tr><td style="width: 25%;"><a href="qbetabinom.html">qbetabinom</a></td>
<td>Quantile for betabin function</td></tr>
<tr><td style="width: 25%;"><a href="qbetabinom.ab.html">qbetabinom.ab</a></td>
<td>Quantile function for the beta-binomial distribution</td></tr>
<tr><td style="width: 25%;"><a href="select.reference.set.html">select.reference.set</a></td>
<td>Combine multiple samples to optimize the reference set in order to maximise the power to detect CNV.</td></tr>
<tr><td style="width: 25%;"><a href="somatic.CNV.call.html">somatic.CNV.call</a></td>
<td>somatic.CNV.call</td></tr>
<tr><td style="width: 25%;"><a href="TestCNV-ExomeDepth-method.html">TestCNV</a></td>
<td>TestCNV</td></tr>
<tr><td style="width: 25%;"><a href="TestCNV-ExomeDepth-method.html">TestCNV-method</a></td>
<td>TestCNV</td></tr>
<tr><td style="width: 25%;"><a href="viterbi.hmm.html">viterbi.hmm</a></td>
<td>Computes the Viterbi path for a hidden markov model</td></tr>
</table>
</div></body></html>
